src/components/Clock/Clock.css

```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Poppins:wght@400;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:wght@500&display=swap");
+.timer {
+  /* width: 100vw; */
+  height:25vh;
+  display: flex;
+  /* flex-direction: column; */
+  justify-content: center;
+}
+.clock {
+  display: flex;
+  flex-direction: row;
+}
+.clock small{
+  color: #FFFFFF;
+  font-size: 15px;
+}
+#days p {
+  color: #FFFFFF;
+  font-size: 2rem;
+  font-family: "Fira Sans", sans-serif;
+}
+#hours p {
+  color: #FFFFFF;
+  font-size: 2rem;
+  font-family: "Fira Sans", sans-serif;
+}
+#minutes p {
+  color: #FFFFFF;
+  font-size: 2rem;
+  font-family: "Fira Sans", sans-serif;
+}
+#seconds p {
+  color: #FFFFFF;
+  font-size: 2rem;
+  font-family: "Fira Sans", sans-serif;
```

Comment:
use poppins instead of fira-sans
Refer to the design first for styling any element